backend/tests/courses/test_friendship_models.py

```diff
@@ -0,0 +1,80 @@
+from django.contrib.auth.models import User
```

Comment:
Sorry ik this is an annoying comment, but usually we want to test code from the API interface, rather than testing specific implementation details (like the model). This makes the tests less brittle / less of "change indicators". It also provides more comprehensive testing, as lots of our interesting logic is in the viewset, not just the model. Issues with the model will surface if we properly test the code from the API level.

The schedules test are a good example of this:
https://github.com/pennlabs/penn-courses/blob/master/backend/tests/plan/test_schedule.py

You can actually send test requests e.g.:
```python
sender = User.objects.create_user(
    username="sender", email="sender@example.com", password="top_secret"
)
recipient = User.objects.create_user(
    username="recipient", email="recipient@example.com", password="top_secret"
)
self.client.login(username="sender", password="top_secret")
response_sent = self.client.post(
    "/api/plan/friendships/",
    json.dumps({"friend_id": recipient.id}),
    content_type="application/json",
)
# TODO: check response_sent for correctness, e.g. status code, content
recipient_client = APIClient()
recipient_client.login(username="recipient", password="top_secret")
response_accepted = recipient_client.put(
    "/api/plan/friendships/",
    json.dumps({"friend_id": sender.id}),
    content_type="application/json",
)
# TODO: check response_accepted for correctness, e.g. status code, content
```